modules/renter/hostdb/consts.go

```diff
@@ -9,11 +9,11 @@ import (
 const (
 	// maxHostDowntime specifies the maximum amount of time that a host is
 	// allowed to be offline while still being in the hostdb.
-	maxHostDowntime = 30 * 24 * time.Hour
+	maxHostDowntime = 10 * 24 * time.Hour
 
 	// minScans specifies the number of scans that a host should have before the
 	// scans start getting compressed.
-	minScans = 20
+	minScans = 12
```

Comment:
hosts only need 12 scans before being pruned. This will help speed up compression and pruning.